MyCryptoHQ/MyCrypto
# Enable Parity Signer Message Signing

### Description

Allows to use the Parity Signer to sign messages :).

### Changes

* The `Sign Message` button now triggers a `SIGN_MESSAGE_REQUESTED` action.
* `IFullWallet.signMessage` is now being used by `sagas/message/signing` for non-Signer wallets.
* Parity Signer wallet handles the flow within the saga and ignores the `signMessage` implementation, analog to `signRawTransaction`.

### Steps to Test

1. Download the Parity Signer from the App Store / Google Play. Create an account on it.
2. Attempt to sign a message with the account on the signer.
3. Verify that the signature is valid.
4. Make sure that signing a message by other means works.
5. (Optional sanity check) check that sending a transaction still checks.

### TODO

* [x] Verify that the signatures are valid (in case the scanned QR code is not a signature or not a valid signature).
